code/edge/src/main/java/com/adobe/marketing/mobile/EdgeNetworkService.java

```diff
@@ -339,7 +342,7 @@ public void call(final HttpConnecting connection) {
 			countDownLatch.await();
 			return httpConnecting[0];
 		} catch (final InterruptedException | IllegalArgumentException e) {
-			Log.warning(LOG_TAG, "Connection failure for url (%s), error: (%s)", url, e);
+			Log.warning(LOG_TAG, LOG_SOURCE, "Connection failure for url (%s), error: (%s)", url, e);
```

Comment:
I've only updated the EdgeNetworkService to fix this one instance of `Log` to get the code to compile. The reset of the class will be updated in another PR.